### PR TITLE
Upversion coredns to latest version 1.8.3

### PR DESCRIPTION
**Pull request by ericgraf**
Release notes: [https://coredns.io/2021/02/24/coredns-1.8.3-release/](https://coredns.io/2021/02/24/coredns-1.8.3-release/)

### CODE REVIEW

**Comment by ericgraf** [comment]
@stone-z  can you show me how you tested the last few coredns releases ?

File: helm/coredns-app/templates/rbac.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ rules:
   - services
   - pods
   - namespaces
+  - endpointslices
```
Comment (stone-z) [inline helm/coredns-app/templates/rbac.yaml:16]: You'll probably have to put this in the `discovery.k8s.io` api group

Comment (ericgraf) [reply helm/coredns-app/templates/rbac.yaml:16]: yep :) thank you for pointing that out.

File: helm/coredns-app/templates/rbac.yaml [hunk 2]
```diff
@@ -17,6 +17,12 @@ rules:
   verbs:
   - list
   - watch
+- apiGroups:
+  - "discovery.k8s.io"
+  resources:
+  - endpointslices
+  verbs:
+  - watch
```
Comment (stone-z) [inline helm/coredns-app/templates/rbac.yaml:24]: Probably still need `list` https://github.com/coredns/deployment/blob/master/kubernetes/coredns.yaml.sed

Comment (ericgraf) [reply helm/coredns-app/templates/rbac.yaml:24]: added

**Comment by stone-z** [comment]
I re-tested this on a `gaia` WC and conformance passed

**Review by stone-z — APPROVED** [review]

**Pull request merged**